Globex-Database/README.md

```diff
@@ -5,11 +5,10 @@
 You will need to have the following tools installed:
 
 * [Docker](https://www.docker.com/products/docker-desktop)
-* A tool to let you interact with an MS SQL database.

```

Comment:
I've removed the original recommendation to install PGAdmin (as we want to try and push learners to using VSCode as it generally requires fewer permissions to install).